packages/nvidia_nat_eval/src/nat/plugins/eval/runtime/evaluate.py

```
@@ -626,6 +635,14 @@ async def run_and_evaluate(self,
                                          adjust_dataset_size=self.config.adjust_dataset_size,
                                          custom_pre_eval_process_function=custom_pre_eval_process_function)
         self.eval_input = dataset_handler.get_eval_input_from_dataset(self.config.dataset)
+        if self.callback_manager and self.eval_input.eval_input_items:
+            try:
+                file_path = getattr(dataset_config, 'file_path', 'nat-eval-dataset')
+                dataset_name = Path(file_path).stem if file_path else 'nat-eval-dataset'
+                self.callback_manager.on_dataset_loaded(dataset_name=dataset_name,
+                                                        items=self.eval_input.eval_input_items)
+            except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)